solutions/08-placement-restricted/movr/cockroach/init_db.sh

```diff
@@ -23,6 +23,11 @@ echo "CONNECTING TO COCKROACH_DB"
 echo "$URL"
 echo "----------------------------------------------------------------------"
 
+echo "----------------------------------------------------------------------"
+echo "UPDATING CLUSTER TO SUPPORT PLACEMENT RESTRICTED "
+echo "----------------------------------------------------------------------"
+cockroach sql --url $URL --execute 'SET CLUSTER SETTING sql.defaults.multiregion_placement_policy.enabled = on;'
```

Comment:
Looks like this is the same command that is being deleted from lines 51-55, just with the new API. Is that right? Just confirming that I'm reading things correctly b/c wtf why would they have `<policy_path>.enabled = on` rather than just `<policy_path> = on`? It doesn't feel wrong exactly but does feel like the Department of Redundancy Department is alive and well in our back-end.

Reply:
Also [the docs](https://www.cockroachlabs.com/docs/stable/data-domiciling.html#step-4-apply-stricter-replica-placement-settings) claim that both syntaxes are legit. Is there any reason you're changing it?

Reply:
So this is a little strange, but there's a `cluster setting` and a `session setting` and we are using the `cluster setting` here which uses the `.enable = on` syntax.

Reply:
Oh! I don't think I was reading the docs carefully enough. Thank you!